src/Automation/Automation/Automation.csproj

```diff
@@ -12,7 +12,9 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.Automation" Version="3.8.3-preview" />
+    <PackageReference Include="Azure.ResourceManager.Automation" Version="1.0.0-beta.1" />
```

Comment:
Our infra hasn't supported track 2 management plane SDKs. For the other options, please contact @dingmeng-xue